releasetool/commands/tag/python_tool.py

```diff
@@ -61,15 +62,21 @@ def create_release(ctx: python.Context) -> None:
 
 
 def publish_to_pypi(ctx: python.Context) -> None:
-    # TODO: Replace this with Kokoro!
-    click.secho("> Publishing to PyPI.")
-    shutil.rmtree("build", ignore_errors=True)
-    shutil.rmtree("dist", ignore_errors=True)
-    for path in glob.glob("*.egg-info"):
-        shutil.rmtree("path", ignore_errors=True)
-    subprocess.check_output(["python3", "setup.py", "sdist", "bdist_wheel"])
-    dists = glob.glob("dist/*")
-    subprocess.check_call(["twine", "upload"] + dists)
+    kokoro_url = "https://fusion.corp.google.com/projectanalysis/current/KOKORO/"
+    project = f"prod:cloud-devrel/client-libraries/{ctx.package_name}/release"
+    project_url = parse.urljoin(kokoro_url, parse.quote_plus(project))
+
+    commitish = ctx.release_pr["merge_commit_sha"]
```

Comment:
Can we use the release tag name as the commitish? It's a bit more human-friendly.

Reply:
That makes sense, done!